src/terrain/routes/schemas/qms.clj

```diff
@@ -169,8 +172,11 @@
   {(optional-key :force)
    (describe String "True if the subscription should be created even if the user already has a higher level plan")})
 
+;; The :periods parameter uses s/Int so that we can get automatic coercion.
 (defschema AddSubscriptionParams
-  {:paid (describe Boolean "True if the user paid for the subscription")})
+  {:paid                    (describe Boolean "True if the user paid for the subscription")
+   (optional-key :periods)  (describe s/Int "The number of subscription periods")
```

Comment:
Coercion wasn't working when I tried to use `java.lang.Integer`. Part of me thinks that we should default to using `s/Int` for schemas, although I didn't make that change.